invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/records/TombstoneForm.js

```diff
@@ -0,0 +1,191 @@
+/*
+ * // This file is part of Invenio-App-Rdm
+ * // Copyright (C) 2023 CERN.
+ * //
+ * // Invenio-App-Rdm is free software; you can redistribute it and/or modify it
+ * // under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import RemovalReasonsSelect from "./RemovalReasonsSelect";
+import { RecordModerationApi } from "../records/api";
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import { Formik } from "formik";
+import {
+  withCancel,
+  ErrorMessage,
+  TextField,
+  TextAreaField,
+} from "react-invenio-forms";
+import { Form, Button, Modal, Divider } from "semantic-ui-react";
+import { NotificationContext } from "@js/invenio_administration";
+import * as Yup from "yup";
+
+export default class TombstoneForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: false,
+      error: undefined,
+    };
+    this.tombstoneSchema = Yup.object({
+      removal_reason: Yup.string().required(),
+      note: Yup.string().required(),
+      citation_text: Yup.string().required(),
+      is_visible: Yup.bool().required(),
+    });
+  }
+
+  componentWillUnmount() {
+    this.cancellableAction && this.cancellableAction.cancel();
+  }
+
+  static contextType = NotificationContext;
+
+  handleSubmit = async (values) => {
+    const { addNotification } = this.context;
+    const { resource, actionSuccessCallback } = this.props;
+    const payload = { ...values };
+
+    this.setState({ loading: true });
+
+    this.cancellableAction = withCancel(
+      RecordModerationApi.deleteRecord(resource, payload)
+    );
+
+    try {
+      await this.cancellableAction.promise;
+      this.setState({ loading: false, error: undefined });
+      addNotification({
+        title: "Success",
+        content: `Record ${resource.id} was deleted.`,
+        type: "success",
+      });
+      actionSuccessCallback();
+    } catch (error) {
+      if (error === "UNMOUNTED") return;
+
+      this.setState({
+        error: error?.response?.data?.message || error?.message,
+        loading: false,
+      });
+      console.error(error);
+    }
+  };
+
+  handleModalClose = () => {
+    const { actionCancelCallback } = this.props;
+    actionCancelCallback();
+  };
+
+  initFormValues = () => {
+    const { resource } = this.props; // TODO take citation
```

Comment:
question for someone more experienced: how can I get the citation from UI record serializer in the search endpoint?

Reply:
do you mean in the search results? Was it forgotten? Or what is the issue?

BTW, see the linter warning above.